tools/cldr-apps/src/main/java/org/unicode/cldr/web/WebContext.java

```diff
@@ -1449,28 +1465,30 @@ public void setSession() {
         }
         String email = field(SurveyMain.QUERY_EMAIL);
 
+        User user = null;
+
         // if there was an email/password in the cookie, use that.
         {
-            String myEmail = getCookieValue(SurveyMain.QUERY_EMAIL);
-            String myPassword = getCookieValue(SurveyMain.QUERY_PASSWORD);
-            if (myEmail != null && (email == null || email.isEmpty())) {
-                email = myEmail;
-                if (myPassword != null && (password == null || password.isEmpty())) {
-                    password = myPassword;
-                }
-            } else {
-                if (myEmail != null && !myEmail.equals(email)) {
-                    removeLoginCookies(request, response);
+            final String jwt = getCookieValue(SurveyMain.COOKIE_SAVELOGIN);
+            if (jwt != null && !jwt.isBlank()) {
+                final String jwtId = CookieSession.sm.klm.getSubject(jwt);
```

Comment:
This is the core usage:  the cookie is decoded, and used to extract a user ID.  It would not validate if the token was compromised (it's cryptographically signed by our secret key, and has an expiration)